metadata.rb

```diff
@@ -3,15 +3,16 @@
 maintainer_email 'chef@osuosl.org'
 issues_url       'https://github.com/osuosl-cookbooks/osl-nginx/issues'
 source_url       'https://github.com/osuosl-cookbooks/osl-nginx'
-license          'Apache 2.0'
+license          'Apache-2.0'
+chef_version     '>= 12.18' if respond_to?(:chef_version)
 description      'Installs/Configures osl-nginx'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '2.1.2'
 
 depends          'certificate'
 depends          'firewall'
 depends          'logrotate'
-depends          'chef_nginx', '~> 6.2.0'
+depends          'nginx'
```

Comment:
Please add a pessimistic lock using the most current version for this since this is a wrapper cookbook for this.